### ISSUE TITLE
theming ref [`color extractor`](http://www.css-color-extractor.com/

### ISSUE THREAD

**Issue by azim254** [opensource254/opensource254.github.io#25]
ref [`color extractor`](http://www.css-color-extractor.com/
)
```
`.plat {
    border-radius: 0;
}

.btn-primary.focus,
.btn-primary:focus {
    box-shadow: 0 0 0 0 #048f83 !important;
}

.btn-primary:hover {
    color: #fff;
    background-color: #016090;
    border-color: #016090;
}

.btn.btn-primary {
    background-color: #048f83 !important;
    border-color: #048f83 !important;
}

span {
    font-style: inherit;
    font-weight: inherit;
}

.features-boxed {
    color: #313437;
    background-color: #eef4f7;
}

.features-boxed p {
    color: #7d8285;
}

.features-boxed h2 {
    font-weight: 700;
    margin-bottom: 40px;
    padding-top: 40px;
    color: inherit;
}

@media (max-width: 767px) {
    .features-boxed h2 {
        margin-bottom: 25px;
        padding-top: 25px;
        font-size: 24px;
    }
}

.features-boxed .intro {
    font-size: 16px;
    max-width: 500px;
    margin: 0 auto;
}

.features-boxed .intro p {
    margin-bottom: 0;
}

.features-boxed .features {
    padding: 50px 0;
}

.features-boxed .item {
    text-align: center;
}

.features-boxed .item .box {
    text-align: center;
    padding: 30px;
    background-color: #fff;
    margin-bottom: 30px;
}

.features-boxed .item .icon {
    font-size: 60px;
    color: #12958a;
    margin-top: 20px;
    margin-bottom: 35px;
}

.features-boxed .item .name {
    font-weight: 700;
    font-size: 18px;
    margin-bottom: 8px;
    margin-top: 0;
    color: inherit;
}

.features-boxed .item .description {
    font-size: 15px;
    margin-top: 15px;
    margin-bottom: 20px;
}

#brand-logo {
    font-size: 34px;
}

#app-navbar {
    background: linear-gradient(120deg, #00e4d0, #5983e8);
}

.navigation-clean-button {
    background: #fff;
    padding-top: 0.75rem;
    padding-bottom: 0.75rem;
    color: #333;
    border-radius: 0;
    box-shadow: none;
    border: none;
    margin-bottom: 0;
}

@media (min-width: 768px) {
    .navigation-clean-button {
        padding-top: 1rem;
        padding-bottom: 1rem;
    }
}

.navigation-clean-button .navbar-brand {
    font-weight: 700;
    color: inherit;
}

.navigation-clean-button .navbar-brand:hover {
    color: #222;
}

.navigation-clean-button .navbar-toggler {
    border-color: #ddd;
    color: #888;
}

.navigation-clean-button .navbar-toggler:focus,
.navigation-clean-button .navbar-toggler:hover {
    background: 0 0;
}

.navigation-clean-button .form-inline,
.navigation-clean-button .navbar-collapse {
    border-top-color: #ddd;
}

.navigation-clean-button .navbar-nav a.active,
.navigation-clean-button .navbar-nav > .show > a {
    background: 0 0;
    box-shadow: none;
}

.navigation-clean-button.navbar-light .navbar-nav a.active,
.navigation-clean-button.navbar-light .navbar-nav a.active:focus,
.navigation-clean-button.navbar-light .navbar-nav a.active:hover {
    color: #8f8f8f;
    box-shadow: none;
    background: 0 0;
    pointer-events: none;
}

.navigation-clean-button.navbar .navbar-nav .nav-link {
    padding-left: 18px;
    padding-right: 18px;
}

.navigation-clean-button.navbar-light .navbar-nav .nav-link {
    color: #465765;
}

.navigation-clean-button.navbar-light .navbar-nav .nav-link:focus,
.navigation-clean-button.navbar-light .navbar-nav .nav-link:hover {
    color: #37434d !important;
    background-color: transparent;
}

.navigation-clean-button .navbar-nav > li > .dropdown-menu {
    margin-top: -5px;
    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
    background-color: #fff;
    border-radius: 2px;
}

.navigation-clean-button .dropdown-menu .dropdown-item,
.navigation-clean-button .dropdown-menu .dropdown-item:focus {
    line-height: 2;
    font-size: 14px;
    color: #37434d;
}

.navigation-clean-button .dropdown-menu .dropdown-item:focus,
.navigation-clean-button .dropdown-menu .dropdown-item:hover {
    background: #eee;
    color: inherit;
}

.navigation-clean-button .actions .login {
    margin-right: 1rem;
    text-decoration: none;
    color: #465765;
}

.navigation-clean-button .navbar-text .action-button,
.navigation-clean-button .navbar-text .action-button:active,
.navigation-clean-button .navbar-text .action-button:hover {
    background: #56c6c6;
    border-radius: 20px;
    color: #fff;
    box-shadow: none;
    border: none;
    text-shadow: none;
    padding: 0.5rem 1rem;
    transition: background-color 0.25s;
    font-size: inherit;
}

.navigation-clean-button .navbar-text .action-button:hover {
    background: #66d7d7;
}

footer {
    background-color: #292c2f;
    box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.12);
    box-sizing: border-box;
    width: 100%;
    text-align: left;
    font: 700 16px sans-serif;
    padding: 55px 50px;
    margin-top: 80px;
    bottom: 0;
    color: #fff;
}

.footer-navigation h3 {
    margin: 0 0 20px;
    font: 400 36px "Cookie", cursive;
    color: #fff;
}

.footer-navigation h3 a {
    text-decoration: none;
    color: #fff;
}

.footer-navigation h3 span {
    color: #5383d3;
}`
```

results
```
#048f83
#016090
#313437
#eef4f7
#7d8285
#12958a
#00e4d0
#5983e8
#465765
#37434d
#56c6c6
#66d7d7
#292c2f
#5383d3
#8f9296
#33383b``
```

ntamalizia hapo chini na naming

_Originally posted by @azim254 in https://github.com/opensource254/opensource254.github.io/pull/22#issuecomment-634233427_

**Comment by stannlee** [comment]
Nice one thanks

**Comment by stannlee** [comment]
closed with #22
